packages/@lwc/integration-karma/test/context/x/advancedProvider/advancedProvider.js

```diff
@@ -62,20 +62,6 @@ function createNewConsumerMeta(consumer) {
     return meta;
 }
 
-function decommissionConsumer(consumer) {
-    const meta = ConsumerMetaMap.get(consumer);
-    if (meta === undefined) {
-        // this should never happen unless you decommission consumers
-        // manually without waiting for the disconnect to occur.
-        throw new TypeError(`Invalid context operation.`);
-    }
-    // take care of disconnecting everything for this consumer
-    // ...
-    // then remove the identity and consumer from maps
-    IdentityMetaMap.delete(meta.identity);
-    ConsumerMetaMap.delete(consumer.identity);
-}
-
```

Comment:
This code was never actually hit. With native lifecycle callbacks, it is hit, and it causes a test error.